vcluster_versioned_docs/version-0.29.0/_partials/config/experimental/isolatedControlPlane.mdx

```diff
@@ -0,0 +1,88 @@
+
```

Comment:
I thought we removed this one, but it exists in main, maybe it got accidentally added back.

Reply:
That is automatically generated from vCluster config so we'd have to talk with @hidalgopl about why it's being created. But it's not published anywhere.